sources/Frontieres.cpp

```diff
@@ -601,6 +602,33 @@ void printParam()
                         myValue.c_str(), 100.0f);
             //            myValue = "Duration (ms): " + theCloud->getDurationMs();
             break;
+        case NUM:
+            myValue = _S("", "Cloud Num: ");
+            sinput << theCloud.getId();
+            myValue = myValue + sinput.str();
+            draw_string((GLfloat)mouseX, (GLfloat)(screenHeight - mouseY), 0.0,
+                        myValue.c_str(), 100.0f);
+            break;
+        default:
+            break;
+        }
+    }
+    if (selectedSound) {
+        SoundRect &theSoundRect = *selectedSound->view;
+        string myValue;
+        ostringstream sinput;
+        ostringstream sinput2;
+        float theA = 0.7f + 0.3 * sin(1.6 * PI * GTime::instance().sec);
+        glColor4f(1.0f, 1.0f, 1.0f, theA);
+
+        switch (currentParam) {
+        case NAME:
+            myValue = _S("", "Sample: ");
```

Comment:
pareil, afin d'éviter `ostringstream`

```suggestion
            myValue = _S("", "Sample: ") + theSoundRect.getName();
```